partners.yml

```diff
@@ -0,0 +1,546 @@
+  
+    event:
+      required: true
+      description: Send event about application
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/event"
+
+  responses:
+    link:
+      description: Successful operation
+      content:
+        application/json:
+          schema:
+            type: object
+            required:
+            - link
+            properties:
+              link:
+                type: string
+                description: Tink link allowing customer to authenticate his bank.
+                example: "https://redirect-here"
+
+    empty:
+      description: Successful operation
+      content:
+        application/json:
+          schema:
+            type: object
+
+    application:
+      description: Successful operation
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/application"
+
+    extendedApplication:
+      description: Successful operation
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/extendedApplication"
+
+    dataError:
+      description: Invalid data
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+  
+    authError:
+      description: Invalid authentication
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+  
+    serverError:
+      description: Internal Server Error. If the problem persist contact the tecnical support
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+  
+    serviceError:
+      description: Service Error. If the problem persist contact the tecnical support
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/error"
+  
+  schemas:
+    application:
+      type: object
+      required:
+      - amount
+      - repaymentTime
+      - reasonForLoan
+      - isBorrowingForOwnUse
+      - applicants
+      properties:
+        referenceId:
+          type: string
+          description: Partner reference id for the application
+        amount:
+          type: number
+          description: Amount requested to borrow in SEK (including refinancing)
+          minimum: 0
+          exclusiveMinimum: true
+          example: 100000
+        repaymentTime:
+          type: number
+          description: Repayment time requested in months
+          minimum: 0
+          exclusiveMinimum: true
+          example: 24
+        reasonForLoan:
+          type: string
+          description: Purpose of the loan. The enum values will be provided to the partner.
+          enum:
+          - vehicle
+          - refinancing
+          - rennovation
+          - ...
+        isBorrowingForOwnUse:
+          type: boolean
+          description: Indicate if customer is asking the loan for himself
```

Comment:
Indicate if customer is applying for a loan for himself or if he is applying on behalf of someone else.